rules/linux/command_and_control_tunneling_via_earthworm.toml

```diff
@@ -20,7 +20,12 @@ license = "Elastic License v2"
 name = "Potential Protocol Tunneling via EarthWorm"
 note = """## Setup
 
-If enabling an EQL rule on a non-elastic-agent index (such as beats) for versions <8.2, events will not define `event.ingested` and default fallback for EQL rules was not added until 8.2, so you will need to add a custom pipeline to populate `event.ingested` to @timestamp for this rule to work.
+If enabling an EQL rule on a non-elastic-agent index (such as beats) for versions <8.2,
```

Comment:
This rule is minstacked to 8.3, is this info for  <8.2 versions relevant?

Reply:
To clean up this we would need this PR to be merged. https://github.com/elastic/detection-rules/pull/3063/files
So this will move as is because we are enhancing the existing configs and once the love PR is merged we will have a PR to strip this information across all rules in a separate PR.